README.md

```diff
@@ -1,2 +1,63 @@
-# Restaurant-page
+# Restaurant-page (Oslo-diner)
 I used what we’ve learned and take a chance to continue practicing DOM manipulation by dynamically rendering a simple restaurant homepage! At the end, we use use JavaScript alone to generate the entire contents of the website!
+ 
+
+![screenshot](./screenshot.png)
```

Comment:
## Fix the screenshot issue.
- [ ] The page's screenshot is not showing in the readme. Kindly fix this. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))